great_expectations/util.py

```diff
@@ -1673,7 +1674,7 @@ def get_context(
     ge_cloud_base_url: Optional[str] = None,
     ge_cloud_access_token: Optional[str] = None,
     ge_cloud_organization_id: Optional[str] = None,
-    ge_cloud_mode: bool = False,
+    ge_cloud_mode: Optional[bool] = None,
```

Comment:
Why are we changing this to be `bool` or `None`?
Maybe I just missed it but don't `True` or `False` cover everything?

Having these 3 values would lead me to believe there are 3 possible branches the code could take, but it looks like there are still just 2.